### DISCUSSION TITLE
Spinner lags my game depending on the skins

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#26776] [Q&A]
Ever since I started playing osu! lazer (in June), I consistently experience a drop in FPS whenever a spinner appears. I test with all skins and my fps drop to :

- 100 fps on Argon
- 160 fps on Triangles
- 210 fps on Classic

Normally I'm on 240 fps.

I've recorded a video testing spinners with all skins with the performance graph. : https://youtu.be/32CULe8cQuQ
Unfortunately, the recording lag my game.

I have included the logs from the recording session and the logs from a non-recording session where I performed the same actions in the video.

[log recording.zip](https://github.com/ppy/osu/files/14085775/log.recording.zip)
[log not recording.zip](https://github.com/ppy/osu/files/14085778/log.not.recording.zip)

My specs
![spec](https://github.com/ppy/osu/assets/101805588/afe1d0b4-39f3-4daa-a11d-dcabdcb3694a)

Can I do something for preventing this ?

**Comment by ghost** [comment]
By modifying the code in a dirty way, it seems that the glow effect is the cause of my game lag. Is there a good way to deactivate the glow?

**Reply by peppy** [reply]
Use classic skin.
